addon/components/horizontal-bar-chart.js

```diff
@@ -451,12 +455,41 @@ const HorizontalBarChartComponent = ChartComponent.extend(FloatingTooltipMixin,
         return this._getElementForValue(valueLabelElements, val);
       });
 
-      const leftLabels = negativeValueLabels.concat(positiveGroupingLabels);
-      const rightLabels = positiveValueLabels.concat(negativeGroupingLabels);
+      const maxNegativeValueLabelWidth = this._maxWidthOfElements(negativeValueLabels);
+      const maxPositiveGroupingLabelWidth = Math.min(maxLabelWidth, this._maxWidthOfElements(positiveGroupingLabels));
+
+      const maxPositiveValueLabelWidth = this._maxWidthOfElements(positiveValueLabels);
+      const maxNegativeGroupingLabelWidth = Math.min(maxLabelWidth, this._maxWidthOfElements(negativeGroupingLabels));
+
+      const minValue = Math.abs(this.get('minValue'));
+      const maxValue = this.get('maxValue');
+      const containerWidth = this.get('outerWidth');
+      const axisTitleOffset = this.get('yAxisTitleHeightOffset') + 5;
```

Comment:
The `5` came from what's used in the drawGraphic method here: https://github.com/Addepar/ember-charts/blob/akhilesh/wip_flush_chart/addon/components/horizontal-bar-chart.js#L550

May be we can turn this into a constant.